packages/core/ios/Fabric/LottieContainerView.h

```diff
@@ -0,0 +1,28 @@
+@class UITraitCollection;
+@class NSDictionary;
+@class NSString;
+@class NSCoder;
+
+#import <React/RCTView.h>
+
+@interface LottieContainerView : RCTView
```

Comment:
Even though this swift class is available in objective-c because we set the `@objc(LottieContainerView)`, we have to create this interface manually because the class is `internal`.

Reply:
Can I mark this as resolved?
Or maybe add your comment in the code so that we can refer back to it.